### DISCUSSION TITLE
new user not register

### DISCUSSION THREAD

**Discussion by biswajit1997** [AxeWP/wp-graphql-headless-login#43] [Q&A] [answered]
How can create a new user

**Comment by biswajit1997** [comment]
existing user login fine but new user not login

**Comment by justlevine** [comment]
Hey @biswajit1997 

Did you make sure to enable the `Login Existing Users` setting on the backend? (See  `4` in the screenshot [on the Settings Overview](https://github.com/AxeWP/wp-graphql-headless-login/blob/develop/docs/settings.md) doc ).

If yes, and it's still not working, please create a [bug report](https://github.com/AxeWP/wp-graphql-headless-login/issues/new?assignees=&labels=&template=bug_report.yml) with enough information for me to replicate, and I'll gladly look into it.

**Comment by biswajit1997** [comment]
actually, issue in google login,
error :----
<img width="938" alt="Screenshot 2023-03-02 at 11 47 07 AM" src="https://user-images.githubusercontent.com/71944996/222346586-523f5419-c95b-47ef-b9c7-ebcedeb8e0e6.png">

**Reply by justlevine** [reply]
Not entirely sure what's going on there solely on the little information you're providing.

In Google, When the hosted domain is set to `*`, it is supposed to match _any_ domain name ([code ref](https://github.com/AxeWP/wp-graphql-headless-login/blob/ff46be69479d5ee27be9f3612e0f1231b774751d/vendor-prefixed/league/oauth2-google/src/Provider/Google.php#LL141).

Can you share the OAuth Response you're providing to the mutation, and your ProviderConfig ( _without_ your clientId/appSecret)?

**Comment by biswajit1997** [comment]
<img width="448" alt="Screenshot 2023-03-02 at 1 19 56 PM" src="https://user-images.githubusercontent.com/71944996/222365041-5d82c893-8f24-4fff-860a-dfd0119a4d89.png">
<img width="938" alt="Screenshot 2023-03-02 at 1 19 36 PM" src="https://user-images.githubusercontent.com/71944996/222365075-94e3d002-546d-4b14-887d-9c9d26478910.png">


<img width="938" alt="Screenshot 2023-03-02 at 1 19 27 PM" src="https://user-images.githubusercontent.com/71944996/222365173-1fb3ad2f-c0cb-49a5-8ad7-3c43e619ce2d.png">

**Reply by justlevine** [reply]
Great thanks! I still need the provider configuration.

Can you:
- head over to the [settings](https://github.com/AxeWP/wp-graphql-headless-login/blob/develop/docs/settings.md), 
- click show advanced on the top right (1 in the screen shot),
- select your Google Provider, and share the config. 
**Don't forget to hide your client ID and secret before sharing**.

Im hoping the solution is as simple as setting the Hosted Domain to an empty string instead of `*`, but I would like to try and reproduce to confirm

**Reply by justlevine** [reply]
Also curious, what's the duplicate type warning you're getting in `extensions.debug` in the GraphQL response?

**Comment by biswajit1997** [comment]
biswajit.pradhan@nettantra.net is working fine but biswajitpradhan120@gmail.com is not working

**Comment by justlevine** [comment, marked as answer]
Ah yes, the `*` value for hosted domain doesn't just "optimize" for Google Workspace, it [_restricts_](https://github.com/thephpleague/oauth2-google/issues/62#issuecomment-449556025) domains to Google Workspace.

Can you set the Hosted Domain setting to empty and see if it works?

**Reply by biswajit1997** [reply]
thanks, bro now working fine 😍😍
